src/metabase/upload.clj

```diff
@@ -226,24 +228,25 @@
                           (try (into []
```

Comment:
I kept it greedy, in the end I decided that lazily considering each row was complexity overkill.